### ISSUE TITLE
Invalid xref in System.Net.Security. AuthenticatedStream

### ISSUE THREAD

**Issue by aik-jahoda** [dotnet/dotnet-api-docs#4763]
In Explicit interface implementation, there is invalid xref: System.IO.Stream.System#IDisposable#Dispose
https://review.docs.microsoft.com/en-us/dotnet/api/system.net.security.authenticatedstream?view=net-5.0&branch=pr-en-us-4758#explicit-interface-implementations

**Comment by carlossanlop** [comment]
@joelmartinez @TianqiZhang @gewarren @BillWagner we are seeing this unexpected warning in some PRs. Example:

https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C1%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202009011251366174-4760%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KqHTYObsop%2FjVWfhrzNWpB2I5Y0e5IjiHqOZRcZqTVo%3D&st=2020-09-01T13%3A06%3A01Z&se=2020-10-02T13%3A11%3A01Z&sp=r


File | Status | Preview URL | Details
-- | -- | -- | --
xml/System.Net.Security/SslStream.xml | Warning | View | Line 0: [Warning] Cross reference not found: 'System.IO.Stream.System#IDisposable#Dispose'.

**Comment by mairaw** [comment]
I'm also seeing weird warnings show up in PRs. /cc @mimisasouvanh 

@carlossanlop maybe someone can file an issue with sitehelp so this can be looked at

**Comment by joelmartinez** [comment]
@mairaw yeah this _looks_ like it should be a valid xref ... pointing to Stream's explicit interface implementation for the Dispose method. @TianqiZhang @herohua ... could this be an issue with the xref lookup service?

**Comment by mairaw** [comment]
Seems like it. This one has broken derived links:
https://docs.microsoft.com/en-us/dotnet/api/system.attribute?view=netcore-3.1

/cc @BillWagner @gewarren

**Comment by aik-jahoda** [comment]
The warning correspond with the rendered page:
https://docs.microsoft.com/en-us/dotnet/api/system.net.security.authenticatedstream?view=netcore-3.1#explicit-interface-implementations

On other side the FileStream has the same base class and the same interface method (same xref) is referenced correctly:
https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream?view=netcore-3.1#explicit-interface-implementations

Both links refers to current live documentation.
